process/block/preprocess/transactions.go

```diff
@@ -1312,7 +1327,7 @@ func (txs *transactions) handleBadTransaction(
 	mbb *miniBlocksBuilder,
 	snapshot int,
 ) {
-	log.Trace("bad tx", "error", err.Error(), "hash", wtx.TxHash)
+	log.Debug("bad tx", "error", err.Error(), "hash", wtx.TxHash)
```

Comment:
**Intended, important.**

If we see these in the log, we have to further optimize the flow.

Reply:
I would like to see only an accumulation of data - not every individual. This can be too much. slowing down processing and everything.

Reply:
OK, will revert.